nservicebus/sagas/analyzers.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Roslyn analyzers for sagas
+summary: Details of the Roslyn analyzers that promote code quality in sagas.
+component: Core
+versions: '[7.7,)'
+reviewed: 2022-03-04
+---
+
+Starting in NServiceBus version 7.7, [Roslyn analyzers](https://docs.microsoft.com/en-us/visualstudio/code-quality/roslyn-analyzers-overview) are packaged with the NServiceBus package that analyze the code in sagas and make suggestions for improvements, right in the editor.
```

Comment:
```suggestion
Starting in NServiceBus version 7.7, [Roslyn analyzers](https://docs.microsoft.com/en-us/visualstudio/code-quality/roslyn-analyzers-overview) are packaged with the NServiceBus package that analyzes the code in sagas and makes suggestions for improvements, right in the editor.
```

Reply:
Nope, Roslyn analyzers here is plural, so noun-verb agreement means "analyze" and "make".